mtools/util/logevent.py

```diff
@@ -205,6 +207,16 @@ def duration(self):
 
         return self._duration
 
+    #SERVER-41349 - get connector from the DNS log line
+    @property
+    def connector(self):
+        line_str = self.line_str
+        if(line_str and 'DNS resolution' in line_str):
+            searchString = "connecting to"
+            self._connector = line_str[line_str.rfind(searchString) + len(searchString) + 1:-12]
```

Comment:
that works but it is not nice and really hard to read :) 

You have used regexes in another part of the code but you can use them here to make it easier

Using () denotes groups and the search will give it directly https://docs.python.org/3/howto/regex.html

try this (example that you'll have to adapt):

```
>>> a = re.search("DNS resolution while connecting to ([\w.]+) took ([\d]+)ms", line_str)
>>> a
<re.Match object; span=(10, 60), match='DNS resolution while connecting to ABC took 5000m>
>>> a.groups()
('ABC', '5000')
```